examples/horovod/tf_hvd_simple.py

```diff
@@ -0,0 +1,125 @@
+# External dependencies
+import argparse
+import glob
+import os
+
+# we can control how much memory to give tensorflow with this environment variable
+# IMPORTANT: make sure you do this before you initialize TF's runtime, otherwise
+os.environ["TF_XLA_FLAGS"] = "--tf_xla_enable_xla_devices"
+# os.environ["CUDA_VISIBLE_DEVICES"] = os.environ.get("OMPI_COMM_WORLD_LOCAL_RANK")
```

Comment:
Since the launch script is handling this now, can we drop this line from the Python script?